index.src.html

```diff
@@ -193,17 +193,26 @@ <h2 id="terms">Key Concepts and Terminology</h2>
       with that <a>environment settings object</a> as their <a for=request>client</a>.
     </dd>
 
-    <dt><dfn>same-origin request</dfn></dt>
+    <dt><dfn>resolved referrer</dfn></dt>
     <dd>
-      A {{Request}} <var>request</var> is a <strong>same-origin request</strong>
-      if <var>request</var>'s <a for=request>origin</a> and the <a for=url>origin</a> of
-      <var>request</var>'s <a for=request>current url</a> are <a lt="same origin">the same</a>.
+      A {{Request}} <var>request</var>'s <strong>resolved referrer</strong> is a <a for=/>URL</a>
+      that is the |referrerSource| variable in the [[#determine-requests-referrer]] algorithm. It is
```

Comment:
How about something like: "The resolved referrer is the URL to which the referrer policy is applied. It may be set on the request directly or derived from request's client." ?

Also imo it would be more natural to have a note in the algorithm itself that says "referrerSource is the request's resolved referrer" than to reference the algorithm's variable here.

Reply:
Thanks, I've made the following changes:
 - Removed this definition
 - Inside "determine request's referrer", when `referrerURL` is computed, I made that an actual `<dfn>` instead of just a variable, and associated it with the request, so it can be referenced outside of this algorithm (i.e., in the individual sections that explain what each policy does). I think this is much more clear and clean

If you prefer a different route, just let me know!